docs-content/general/4_company/product-philosphy.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Our Product Philsophy
+slug: product-philosphy
+---
+
+## Overview 
+
+This doc acts as a reference for our product philosophy at highlight.io, or to be more exact, how we think about what we build (not necessarily how we build it). It acts a way for our team to prioritize work, but also educate the public on where we're headed. Sharing this type of content aligns with our values to [build in public](./1_values.md#we-build-in-public) and we hope to continue sharing more in this space.
+
+## Cohesion
+
+Our product philsophy at highlight.io is centered around the concept of "cohesion", or the idea that we're focused on building  a tightly coupled suite of tools that helps developers ship software with confidence.
+
+Prior to working on highlight.io, we all worked at several tech companies of varying sizes, and had first-hand experience trying to stich together numerous tools to reproduce bugs. It wasn't uncommon that we had to, for e.g., login to Sentry to see a stacktrace, log into Splunk to query logs, and after loads more tools, finally succumb to having to login "as the user" to try and reproduce the issue.
```

Comment:
`and after loads more tools, finally succumb to having to login` sounds awkward. maybe `and after switching between a handful of tools, give up and log in "as the user"`...